CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+#					CONTRIBUTING
+
+Contributing to the official buyTogether Application
+Welcome! We're so excited that you're considering contributing to the official GitHub page for buyTogether.
+
+---
+
+## Submitting a Pull Request:
+ - Fork and clone the repository
+ - Create a new branch called X
+ - git checkout X
+ - Push to your fork and submit a pull request.
+
+## Your PR is merged!
+Congratulations.
+
+Once your PR is merged, your contributions will be publicly visible on the GitHub.
+
+---
+
+## Issues:
+### Create a new issue:
+- If you spot a problem, search if an issue already exists. If a related issue doesn't exist, you can open a new issue using a relevant issue form.
+- Make sure what you are describing is the issue.
+- Screenshots are very helpful in diagnosing issues; please include them whenever possible.
+
+---
+
+## **Names** :
+Elipaz Menachem, 28 years old, third year of computer science degree.‏
+Lior Silberman,  27 years old.I'm a third year computer science student
+Meitar Rizner, 26. A third year computer science student.
+Rawan Abuleil , 23 years old, third year of computer science degree
+Karam Jbareen, 23 years old, fourth year of computer science student studying at Tel-Hai.
```

Comment:
Before you push markdown content, you need to preview the rendered file for yourself and examine the result.
Many editors can do that for you, there are online utilities that can do that for you, and even GitHub does that for you (open the file in GitHub and it will preview it for you).
Specifically here, single newline in markdown doesn't add a line break on the rendered file, so all this text is actually in a single line.

Also, please make sure the format for each student is the same.

Lastly, I don't think this list belongs inside this document. It should be a part of your presentation but not a part of a document that explains how to contribute to the project.